src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -0,0 +1,198 @@
+  packages: EventPackageData,
+  device: EventDevice,
+};
+
+class EventDetails extends React.Component {
+  static propTypes = {
+    api: PropTypes.object,
+    eventSlug: PropTypes.string.isRequired,
+  };
+
+  state = {
+    loading: true,
+    error: false,
+    event: null,
+    activeTab: null,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.eventSlug != this.props.eventSlug) {
+      this.fetchData();
+    }
+  }
+
+  async fetchData() {
+    this.setState({loading: true, error: false});
+    const {orgId} = this.props.params;
+    const [projectId, eventId] = this.props.eventSlug.split(':');
+    try {
+      if (!projectId || !eventId) {
+        throw new Error('Invalid eventSlug.');
+      }
+      const response = await this.props.api.requestPromise(
+        `/projects/${orgId}/${projectId}/events/${eventId}/`
+      );
+      this.setState({
+        activeTab: response.entries[0].type,
+        event: response,
+        loading: false,
+      });
+    } catch (e) {
+      this.setState({error: true});
+    }
+  }
+
+  handleClose = event => {
+    event.preventDefault();
+    browserHistory.goBack();
+  };
+
+  handleTabChange = tab => this.setState({activeTab: tab});
+
+  renderBody() {
+    if (this.state.loading) {
+      return <LoadingIndicator />;
+    }
+    if (this.state.error) {
+      return <NotFound />;
+    }
+    const {event, activeTab} = this.state;
+
+    return (
+      <React.Fragment>
+        <EventOrGroupHeader
+          params={this.props.params}
+          data={this.state.event}
+          includeLink={false}
+        />
+        <NavTabs underlined={true}>
+          {event.entries.map(entry => {
+            if (!INTERFACES.hasOwnProperty(entry.type)) {
+              return null;
+            }
+            const type = entry.type;
+            const classname = type === activeTab ? 'active' : null;
+            return (
+              <li key={type} className={classname}>
+                <a
+                  href="#"
+                  onClick={() => {
+                    this.handleTabChange(type);
+                  }}
+                >
+                  {utils.toTitleCase(type)}
+                </a>
+              </li>
+            );
+          })}
+          {Object.keys(OTHER_SECTIONS).map(section => {
+            if (utils.objectIsEmpty(event[section])) {
+              return null;
+            }
+            const classname = section === activeTab ? 'active' : null;
+            return (
+              <li key={section} className={classname}>
+                <a
+                  href="#"
+                  onClick={() => {
+                    this.handleTabChange(section);
+                  }}
+                >
+                  {utils.toTitleCase(section)}
+                </a>
+              </li>
+            );
+          })}
+        </NavTabs>
+        {this.renderActiveTab(event, activeTab)}
```

Comment:
I would wrap with `app/components/errorBoundary` in case something deep down the tree fails to render.